pyat/at/load_mat.py

```diff
@@ -7,20 +7,19 @@
 from .lattice import elements
 import numpy
 
-
 CLASS_MAPPING = {'Quad': 'Quadrupole', 'Sext': 'Sextupole', 'AP': 'Aperture',
                  'RF': 'RFCavity', 'BPM': 'Monitor'}
 
-CLASSES = set(['Marker', 'Monitor', 'Aperture', 'Drift', 'ThinMultipole',
-               'Multipole', 'Dipole', 'Bend', 'Quadrupole', 'Sextupole',
-               'Octupole', 'RFCavity', 'RingParam', 'M66', 'Corrector'])
+CLASSES = {'Marker', 'Monitor', 'Aperture', 'Drift', 'ThinMultipole', 'Multipole', 'Dipole', 'Bend', 'Quadrupole',
+           'Sextupole', 'Octupole', 'RFCavity', 'RingParam', 'M66', 'Corrector'}
```

Comment:
This change is proposed by PyCharm for efficiency: it does not require any function call. Since it's executed only once at import, I guess we are not much concerned by efficiency, so I'll turn it back to set([...]).